hotkey.wiiu and payloadfail.wiiu Commands (#1381)
nh-server/Kurisu: cogs/assistance-cmds/payloadfail.wiiu.md

---
title: Payload failing to load fixes
help-desc: Potential fixes for when payload.elf fails to load on the Wii U
---

**Fixes for "Failed to load payload.elf"**

> - Make sure you have both a `payload.elf` and `payload.rpx` file on your SD card in the `wiiu` folder. Make sure your SD card layout is the same as [this](https://wiiu.eiphax.tech/sdlayout)

Reviewer: ```suggestion
- Make sure you have both a `payload.elf` and `payload.rpx` file in the `wiiu` folder on the SD card. Make sure your SD card layout is the same as shown [here](https://wiiu.eiphax.tech/sdlayout).

```

> - Redownload the payloads from https://aroma.foryour.cafe/. Extract them onto your SD card, merge the Wii U folder. If prompted to replace the existing ones, do so.

Reviewer: ```suggestion
- Redownload the payloads from https://aroma.foryour.cafe/. Extract the contents of the `.zip` file to the root of your SD card, and merge with the existing `wiiu` folder. When prompted to replace the existing ones, do so.

```

> - Make sure your SD card is properly formatted to Fat32. Type .sdformat for a guide on how to properly format your SD card to Fat32.

Reviewer: ```suggestion
- Make sure your SD card is properly formatted to FAT32. A guide on how to do so can be found [here](https://wiki.hacks.guide/wiki/Formatting_an_SD_card).

```

> - Make sure your SD card and SD card slot are clean. Power off your console, remove the SD card, and ensure that there is no dust or dirt obstructing the SD card's contacts or the SD card slot.

Reviewer: ```suggestion
- Make sure your SD card and SD card slot are clean. Power off your console, remove the SD card, and ensure that there is no dust or dirt obstructing the SD card's contacts or the SD card slot.
- Try saving a Mii to the SD card in Mii Maker. ([Guide](https://en-americas-support.nintendo.com/app/answers/detail/a_id/1722/~/how-to-save-a-mii-as-a-photo))
```

- Make sure your SD card is properly inserted into the SD cart slot. You should hear a click sound when it is inserted.

> If none of the above works, you may have a faulty/worn SD card, MicroSD card, or MicroSD card adapter, and are advised to purchase a new SD card or adapter. Type .sdreq for SD card size recommendations. 

Reviewer: ```suggestion
If none of the above works, you may have a faulty/worn SD card, MicroSD card, or MicroSD card adapter, and are advised to purchase a new SD card or adapter. Type `.sdreq` in <#261581918653513729> for SD card size recommendations. 
```
